packages/Discretization/test/PointCloudProblemGenerator/create_ascii_data_from_exodus.py

```diff
@@ -0,0 +1,90 @@
+############################################################################
+# Copyright (c) 2012-2017 by the DataTransferKit authors                   #
+# All rights reserved.                                                     #
+#                                                                          #
+# This file is part of the DataTransferKit library. DataTransferKit is     #
+# distributed under a BSD 3-clause license. For the licensing terms see    #
+# the LICENSE file in the top-level directory.                             #
+############################################################################
+
+# This script reads an exodus file and dumps its node coordinates and
+# connectivities of all elements in all element blocks into 2 seperate files
+# to be used by driver test code
+
+import netCDF4
+import numpy
+import sys
+
+# exodus file
+exodus_file = sys.argv[1]
+
+# prefix of output files
+output_prefix = sys.argv[2]
+
+# load the exodus file
+nc = netCDF4.Dataset( exodus_file )
+
+# extract the node coordinates
+x_coords = nc.variables['coordx']
+y_coords = nc.variables['coordy']
+z_coords = nc.variables['coordz']
+
+# get the number of nodes
+num_node = nc.dimensions['num_nodes'].size
+
+# get the number of element blocks
+num_block = nc.dimensions['num_el_blk'].size
+
+# get the total number of cell
+total_num_cell = nc.dimensions['num_elem'].size
+
+# write out info.
+print "Extracting Exodus file to ASCII. Mesh stats:"
+print "Number of nodes:", num_node
+print "Number of cells:", total_num_cell
+print "Number of blocks:", num_block
```

Comment:
This will be invalid in Python3. Please use the `print()` function rather than a `print` statement.